coffee/classes/mds_config.coffee

```diff
@@ -11,7 +11,7 @@ class MdsConfig
 
   @initialConfig:
     editor:
-      fontFamily: 'Osaka-mono, "MS Gothic", monospace'
+      fontFamily: 'Consolas, monaco, monospace'
```

Comment:
I want to discuss this.
I removed Japanese font from editor's font, and added monospace fonts: `Consolas` for Windows, and `Monaco` for Mac.
However, only `monospace` would work well from the point of internationalize.